lib/plugins/requestExpiry.js

```diff
@@ -38,31 +38,45 @@ function requestExpiry(options) {
     var timeoutHeaderKey = options.timeoutHeader;
 
     return function requestExpirationCheck(req, res, next) {
-        var expiryTime;
+        /*
+         * Add check expiry API to to req if it doesn't already exist. We only
+         * add this the first time this handler is run, since this handler
+         * could be used in multiple places in the handler chain.
+         */
+        if (!req._expiry) {
```

Comment:
Don't know that we've been consistent about appending these directly on req. Is intention to have audit log consume this at some point? If so, then this is probably okay. If not, just use the closure?